server/schedulers/scatter_range.go

```diff
@@ -75,30 +74,18 @@ func (l *scatterRangeScheduler) IsScheduleAllowed(cluster schedule.Cluster) bool
 	return l.opController.OperatorCount(schedule.OpRange) < cluster.GetRegionScheduleLimit()
 }
 
-func (l *scatterRangeScheduler) getOperators(opInfuence schedule.OpInfluence) []*schedule.Operator {
```

Comment:
Is it correct to remove this? /cc @nolouch

Reply:
The rest LGTM. PTAL @nolouch

Reply:
Yes, It is correct. `GetOpInfluence` handle it.